File: Herencia/ejercicio 4/Java/Fijo.java

class TFijo extends TEmpleado {
    public double sueldo;
    public TFijo(String nombre, double sueldo) {
        super(nombre);
        this.sueldo = sueldo;
    }
    public void setSueldo(double sueldo) {
        this.sueldo = sueldo;
    }
    @Override
    public double getSalario() {
        return sueldo;
    }
}

File: Herencia/ejercicio 4/Python/Fijo.py
from Empleado import TEmpleado
class TFijo(TEmpleado):
    def __init__(self, nombre, sueldo):
        super().__init__(nombre)
        self.sueldo = sueldo
    def setSueldo(self, sueldo):
        self.sueldo = sueldo
    def getSalario(self):
        return self.sueldo